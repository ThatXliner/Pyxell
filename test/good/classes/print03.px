
class Test def
    func toString() String def
        return "test"

print [Test()]
print "{Test(), Test().toString()}"

Test? t = null
t = Test()
print t, t!
