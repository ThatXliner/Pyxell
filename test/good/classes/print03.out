[<Test object>]
(<Test object>, "test")
<Test object> test
